src/commands/gitleaks.yaml

```diff
@@ -1,113 +1,28 @@
 parameters:
-  branch:
-    description: "The branch to audit."
-    type: "string"
-  commit:
-    description: "Scan all files changed by this commit (rev. SHA)."
-    type: "string"
-  commit-from:
-    description: "Commit to start audit from (rev. SHA). This is the most recent commit to audit. Default is the latest."
-    type: "string"
   commit-to:
     description: "Commit to stop audit (rev. SHA). This is the oldest commit to audit. Default is the root commit. \
     In CircleCI, you can use << pipeline.git.base_revision >> to scan only since the parent commit."
     type: "string"
   config:
     description: "Path to a config file to use for gitleaks configuration."
     type: "string"
-  debug:
-    description: "If true, will print debug messages."
-    type: "boolean"
-  depth:
-    description: "Number of commits to audit."
-    type: "integer"
   files-at-commit:
```

Comment:
My understanding is this flag allows to specify commit sha to check. What is the use case to check only fixed commit in the CI setup?

Reply:
I'm anticipating noise. I might not want to hear about every high-entropy thing from all time when I make some small change

Reply:
I don't get that. Does that allow to specify files to check or commit to check?

Reply:
It is a commit hash, but it scans all files at the state of that commit, not just the files which were modified. The default behavior would show results for every commit ever. I was thinking this would be more applicable for pushes to branches and master can scan the whole history. That way it isn't noisy for people during daily tasks and we can work on tuning the rules for master

Reply:
I wonder:

- What is the workflow to change that parameter. Should we do this in every PR? That doesn't sound very efficient.
- How useful is having those full reports in the master branch? IMO the job is useful only if it fails when something is wrong otherwise I can guarantee you it will be ignored in 99% cases.

Reply:
I will remove this as well. In the absence of using different configs for branches or different logic this isn't useful.